parts/kubernetesmaster-cloud-controller-manager.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: "v1"
+kind: "Pod"
+metadata:
+  name: "cloud-controller-manager"
+  namespace: "kube-system"
+  labels:
+    tier: control-plane
+    component: cloud-controller-manager
+spec:
+  hostNetwork: true
+  containers:
+    - name: "cloud-controller-manager"
+      image: "<kubernetesCcmImageSpec>"
+      command: 
+        - "cloud-controller-manager"
+        - "--kubeconfig=/var/lib/kubelet/kubeconfig"
+        - "--allocate-node-cidrs=<allocateNodeCidrs>"
+        - "--cluster-cidr=<kubeClusterCidr>"
+        - "--cluster-name=<masterFqdnPrefix>"
+        - "--cloud-provider=azure"
+        - "--cloud-config=/etc/kubernetes/azure.json"
+        - "--leader-elect=true"
+# TODO: RBAC support
```

Comment:
What's needed or outstanding for enabling RBAC?

Reply:
This is actually meant for enabling 'use-service-account-credentials', which does not work now, pending on kubernetes#53511